environment.ts

```diff
@@ -110,6 +110,15 @@ const environment = {
 
   // Keywords of the marketplace to place in the SEO keywords
   META_KEYWORDS: 'NFT, marketplace, platform, blockchain, liteflow',
+
+  /**
+   * NFT Mint Behavior
+   */
+  // Enable/disable the lazy minting feature. If enabled, the NFTs will be minted on the first sale
+  LAZYMINT: true,
+
+  // Enable/disable the unlockable content feature. If enabled, the NFTs will have unlockable content only accessible to owners
+  UNLOCKABLE_CONTENT: true,
```

Comment:
Preserve previous defaults

```suggestion
  LAZYMINT: false,

  // Enable/disable the unlockable content feature. If enabled, the NFTs will have unlockable content only accessible to owners
  UNLOCKABLE_CONTENT: false,
```